core/kazoo_csv/test/kz_csv_tests.erl

```diff
@@ -213,26 +218,26 @@ mapped_row_to_iolist_test_() ->
 
 mapped_row_data() ->
     [{5, <<",,,,\n">>, #{}}
-    ,{5, <<"\"a\",\"b\",,,\n">>, #{<<"1">> => <<"a">>, <<"2">> => <<"b">>}}
-    ,{2, <<"\"a\",\"b\"\n">>, #{<<"1">> => <<"a">>, <<"2">> => <<"b">>}}
-    ,{3, <<"\"a\",,\"b\"\n">>, #{<<"1">> => <<"a">>, <<"2">> => ?ZILCH, <<"3">> => <<"b">>}}
-    ,{2, <<"\"a\",\n">>, #{<<"1">> => <<"a">>, <<"2">> => ?ZILCH, <<"3">> => <<"b">>}}
-    ,{3, <<",,\"b\"\n">>, #{<<"1">> => ?ZILCH, <<"2">> => ?ZILCH, <<"3">> => <<"b">>}}
+    ,{5, <<"a,b,,,\n">>, #{<<"1">> => <<"a">>, <<"2">> => <<"b">>}}
```

Comment:
Why removing the quotes? I believe they are there because some programs export fields wrapped in single or double quotes. Why not add unquoted versions to make sure we're getting the same results regardless of the quoted-ness of the cell?

Reply:
Ok, sorry i missed the reason for failure. Il look in to that. 

In relation to wrapping or not wrapping all cells in double quotes, i dont mind. Both are acceptable ways of doing it. I originally had 2 code paths for simple unwrapped cells vs more complex wrapped cells but then both ended up having the same order N complexity so decided to just drop the simple cell case but kept the not needing to wrap the cell. 

On 4.3 its older implementation dose not wrap every cell and I noticed that you added the wrapping every cell to your master but thought you just added it to help escape some charters / edge case. If you would prefer i can add it back in? The joys of there being no official spec of CSVs 🙄

Reply:
Yeah every time I touch CSV parsing I run away vowing never to touch it again LOL.

If the tests include it then we need to keep it because it was likely added in response to a support ticket.

Reply:
Yeah haha! Well hopefully after this its the last for awhile!! 

Ok yeah i can re-add 👍